src/middlewared/middlewared/plugins/iscsi_/alua.py

```diff
@@ -281,6 +251,27 @@ async def standby_after_start(self, job):
             return
         else:
             job.set_progress(22, 'Remote iscsitarget is active')
+            self.logger.debug('Remote iscsitarget is active')
+
+        # Next turn off cluster_mode for all the extents.
+        # this will avoid "ignore dlm msg because seq mismatch" errors when we reconnect
+        # Rather than try to execute in parallel, we will take our time
+        while self.standby_starting:
+            try:
+                devices = await self.middleware.call('failover.call_remote', 'iscsi.scst.cluster_mode_devices_set')
```

Comment:
When making a `call_remote` call and the TCP session is still open for the remote side but the remote side was abruptly terminated or died (panic'ed), this can hang for awhile. Best to add some arguments to the `call_remote` call here. Please see `mismatch_disks` method for the arguments that I use to be passed to `call_remote`.